example/migrations/src/main/scala/SampleCodegen.scala

```diff
@@ -17,20 +17,22 @@ object SampleCodegen{
         H2Driver.defaultTables.filter(t =>
           tableNames.contains(t.name.name))))
       val latest = mm.latest
-      List( "v" + latest, "latest" ).foreach{ version =>
-        val pkg = "datamodel." + version + ".schema"
-        val folder = System.getProperty("user.dir")+"/app/src/main/scala"
-        val generator = new SourceCodeGenerator(model) {
-          override def packageCode(
-            profile: String, pkg: String, container: String) : String =
-            super.packageCode(profile, pkg, container) + s"""
+      List("/app", "/migrations").foreach { proj =>

```

Comment:
Yes I generate the stuff into two folders. It's just a temporary workaround.

What do you mean? Generating a distinct project and have the application project and migrations project depend on it?